templates/web/base/report/new/form_report.html

```diff
@@ -49,12 +49,13 @@ <h2 class="form-section-heading">[% loc( 'Public details' ) %]</h2>
 [% TRY %][% PROCESS 'report/new/after_photo.html' %][% CATCH file %][% END %]
 
     [% DEFAULT form_detail_label = loc('Explain what’s wrong') %]
+    [% DEFAULT form_detail_placeholder = loc('e.g. ‘This pothole has been here for two months and…’') %]
     <label for="form_detail">[% form_detail_label %]</label>
+    <p class="form-hint" id="detail-hint">[% form_detail_placeholder %]</p>
```

Comment:
Also spacing feels a bit wide around the new examples, as now it's equal between label, examples, text input and next label. Feels like should be bit smaller in the same-thing bits.

Reply:
Fixed!

![screen shot 2018-08-01 at 10 24 32](https://user-images.githubusercontent.com/739624/43513229-232f39ca-9575-11e8-95b5-9204e4cfece4.png)